apps/client/src/components/AddressData.js

```diff
@@ -1,58 +1,60 @@
 import React from "react";
 import { Paragraph } from "@datapunt/asc-ui";
-import AddressResult from "./AddressResult";
+import { getRestrictionByTypeName } from "../utils";
 import { List, ListItem } from "../atoms";
 import { uniqueFilter } from "../utils";
+import { StyledAddressData } from "./AddressDataStyles";
 
-const getRestrictionByTypeName = (address, typeName) =>
-  address.restrictions.find(({ __typename }) => __typename === typeName);
-
-const AddressData = ({ address }) => {
-  const monument = getRestrictionByTypeName(address, "Monument")?.name;
-  const cityScape = getRestrictionByTypeName(address, "CityScape")?.name;
-  const zoningPlans = address.zoningPlans
+const AddressData = ({ address, displayZoningPlans }) => {
+  const { restrictions, zoningPlans } = address;
+  const monument = getRestrictionByTypeName(restrictions, "Monument")?.name;
+  const cityScape = getRestrictionByTypeName(restrictions, "CityScape")?.name;
+  const zoningPlanNames = zoningPlans
     .map((plan) => plan.name)
     .filter(uniqueFilter); // filter out duplicates (ie "Winkeldiversiteit Centrum" for 1012TK 1a)
 
   return (
-    <div>
-      <AddressResult title="Monument:">
-        <Paragraph>
-          {monument ? `Ja. ${monument}` : "Nee. Geen monument"}
-        </Paragraph>
-      </AddressResult>
+    <StyledAddressData>
+      <Paragraph strong style={{ marginBottom: "0px" }}>
```

Comment:
`gutterBottom={0}` not working? Otherwise `{{ marginBottom: 0 }}` or `{{ marginTop: 20 }}` also works without making it a string

Reply:
will fix.